src/App.js

```diff
@@ -7,16 +7,25 @@ class App extends Component {
   constructor() {
     super()
     this.state= {
-      movies: []
+      movies: [],
+      error: ''
     }
   }
 
+  componentDidMount() {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(response => response.json())
+      .then(data => this.setState({movies: data.movies}))
+      .catch(error => this.state.error = 'YA DONE MESSED UP AY-AY-RON!')
+  }
+
   render() {
     return(
-      <div className="App">
+      <main className="App">
         <Header />
+        {this.state.error && <h2>{this.state.error}</h2>}
```

Comment:
I want to understand this line a little better. It's a nice and clean way of error handling though!